src/github.com/getlantern/flashlight/app/mock_autoupdate.go

```diff
@@ -0,0 +1,15 @@
+// +build mockupdate
+
+package app
+
+// This is a public key derived from a private key that is freely available.
+// Don't use in production.
+const packagePublicKey = `-----BEGIN PUBLIC KEY-----
```

Comment:
Which branch were you grepping @xiam ? I couldn't find below line on `pro-backend` branch.

./github.com/getlantern/flashlight/app/app.go:  autoupdate.PublicKey = []byte(packagePublicKey)